RELEASE-NOTES.txt

```diff
@@ -1,3 +1,7 @@
+Unreleased
+------
+* [***] Audio block now available on WP.com sites on the free plan. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3523]
```

Comment:
I moved this entry to the unreleased section as this PR won't be included in `1.54.0`.